Gibe.DittoProcessors/Processors/StringToMd5HashAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Text;
+using System.Security.Cryptography;
+
+namespace Gibe.DittoProcessors.Processors
+{
+	public class StringToMd5HashAttribute : TestableDittoProcessorAttribute
+	{
+		public override object ProcessValue()
+		{
+			if (Value == null) throw new NullReferenceException();
+
+			var stringValue = Value as string;
+
+			// byte array representation of that string
+			byte[] encodedValue = new UTF8Encoding().GetBytes(stringValue);
+
+			// need MD5 to calculate the hash
+			byte[] hashValue = ((HashAlgorithm)CryptoConfig.CreateFromName("MD5")).ComputeHash(encodedValue);
+
+			// string representation (similar to UNIX format)
+			return BitConverter.ToString(hashValue).Replace("-", string.Empty).ToLower();
```

Comment:
Not sure - found this method online and it worked for converting an email into MD5 hash for Gravatar (which is the use case)